boba-chain-ops/genesis/config.go

```diff
@@ -210,6 +212,17 @@ func (d *DeployConfig) Check() error {
 	if d.L2GenesisBlockBaseFeePerGas == nil {
 		return fmt.Errorf("%w: L2 genesis block base fee per gas cannot be nil", ErrInvalidDeployConfig)
 	}
+	// l1 Boba token address is optional, if not provided, use the default address for the chain ID
+	// but if provided, it must be a valid address
+	var l1BobaTokenAddress common.Address
+	if d.L1BobaTokenAddress != nil {
+		l1BobaTokenAddress = *d.L1BobaTokenAddress
+	} else {
+		l1BobaTokenAddress = common.HexToAddress(chain.GetBobaTokenL1Address(big.NewInt(int64(d.L2ChainID))))
+	}
+	if l1BobaTokenAddress == (common.Address{}) {
```

Comment:
This block looks a lot like the one below, but it doesn't seem like we're actually storing/returning the value of `l1BobaTokenAddress` (below it's stuck into a map).  Is that intentional?

Reply:
Yes, it's for the local testing purpose. For local testing, I don't know what the L1 BOBA token address looks like. After deploying contracts, I add the token address to the configuration.json file so that it can generate the right storage for BOBA token. I can subtract this code to a function.